tailwind.config.js

```diff
@@ -79,11 +79,11 @@ module.exports = {
   variants: {
     accessibility: ['responsive', 'focus', 'hocus'],
     appearance: [],
-    backgroundColor: ['focus', 'hover', 'hocus', 'group-hover'],
+    backgroundColor: ['focus', 'hover', 'hocus', 'group-hocus'],
```

Comment:
unless this is magic idk about, I think you want to do a find-and-replace-all 'hocus' -> 'focus' 😛